.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,14 +3,28 @@ Thank you for your pull request! Please review the requirements below
 and ensure your pull request has fulfilled all requirements outlined in the target package.
 -->
 
+## PR checklist
+
+For every PR, make sure that these statements are true:
+- [ ] Includes only changes relevant to the original ticket. Significant refactoring or formatting needs to be separated.
+- [ ] Doesn't contain unnecessary type casts and non-null asserts
+- [ ] Doesn't add `@ts-ignore`
+- [ ] Doesn't disable lints, particularly `no-any` lints
+- [ ] Doesn't use `useState` just for computation - use plain variables instead
+- [ ] Splits components into pure components that don't depend on external state or hooks.
+- [ ] Doesn't add components inside of components that depend on variables in the parent scope - separate them out
+- [ ] Doesn't propagate optional values without good reason, doesn't mark property values as optional if that doesn't represent reality
+- [ ] Doesn't duplicate existing code
+- [ ] Parses out-of-domain data
+- [ ] Does things as simple as possible, but not simpler.
```

Comment:
This one would be hard to judge and easy to argue. I would stick to concrete and objective criteria for this list.